modules/userId/index.js

```diff
@@ -488,6 +488,7 @@ function idSystemInitializer({delay = GreedyPromise.timeout} = {}) {
     }
     cancel = defer();
     return GreedyPromise.race([promise, cancel.promise])
+      .catch(() => logWarn(`${MODULE_NAME} - cancelled promise`))
```

Comment:
as discussed offline, I am not able to repro the issue so I am grasping at straws here - instead of this, could you try replacing 

```
    if (cancel != null) {
      cancel.reject(INIT_CANCELED);
    }
```

with

```
    ((c) => {
      setTimeout(() => {
        if (c != null) {
          c.reject(INIT_CANCELED);
        }
      })
    })(cancel);
```

and see if it makes any difference? (this change also fails tests but I think that's a problem with the tests so if it works I can go ahead and look at them as well).

Reply:
Doesnt appear that worked, if we change from logWarn to logError is that more palatable?

Reply:
@dgirardi @patmmccann Not sure if it's helpful, but I experienced this uncaught promise error when calling `refreshUserIds` twice in quick succession. I was able to eliminate the error by removing one of those calls (it wasn't necessary anymore). It was on my to-do list to raise this up in a Git issue to have the promise fail more gracefully or provide some sort of error context. I was able to reproduce consistently in Chrome. I can probably dig up an old bundle if that's helpful, let me know.